### DISCUSSION TITLE
Show only once ?

### DISCUSSION THREAD

**Discussion by Orsucciu** [AmauriC/tarteaucitron.js#1034] [Q&A]
Hello,

I was tweaking with the settings, and it appeared to me that we can either display the cookie selection permanently or not at all. Is it possible to show it once instead ? Once users have set their preferred policy, we should dismiss the selection button right ?

Thanks

**Comment by AmauriC** [comment]
Hi

The main banner is displayed until the user have made a choice.

The small banner is always displayed but there is a setting if you don't want it.
